HTML/Home.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- meta data -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- this is for icons -->
+    <script
+      src="https://kit.fontawesome.com/52e67ff79e.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- This is for linking Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Actor&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Linking CSS -->
+    <link rel="stylesheet" href="..\CSS-Folder\Home.css" />
+
+    <!-- Tile Name -->
+    <title>Cinema App</title>
+  </head>
+  <body>
+    <!-- Home Header St ------------------------------------------->
+
+    <img class="header-img" src="..\Images\image-12.png" />
+    <rect class="Gradient_Rect"></rect>
+
+    <section class="Header-Spacebox"></section>
+    <section class="Header-Home">
+      <h1>Bohemian Rhapsody</h1>
+      <h2>Drama, Music</h2>
+      <a >Description</a>
+      <!-- <a href=Movie_info.html>Description</a> -->
+    </section>
+
+    <!-------------------------------------------Home Header End -->
+
+    <!-- Reaction Section St. --------------------------------- -->
+    <section class="reaction-section">
+      <a class="Download_icon" > <i class="fas fa-arrow-down"></i> </a>
+      <a class="Love_icon" > <i class="far fa-heart"></i> </a>
+      <a class="Share_icon" > <i class="fas fa-share"></i> </a>
+    </section>
+
+    <!-------------------------------------Reaction Section End -->
+
+    <!-- St of Rating Box ----------------------------------------- -->
+    <section class="Rating-Box">
+        <article class="Top_Rating">
+        <div class="Top_Rating_Stars">
```

Comment:
please use semantic  Html tags for more accessibility 
so, use ul instead of div here